components/event-publisher-proxy/pkg/legacy-events/legacy_unit_test.go

```diff
@@ -103,9 +104,9 @@ func TestTransformLegacyRequestsToCE(t *testing.T) {
 			writer := httptest.NewRecorder()
 			app := applicationtest.NewApplication(tc.givenApplication, applicationTypeLabel(tc.givenTypeLabel))
 			appLister := fake.NewApplicationListerOrDie(ctx, app)
-			transformer := NewTransformer("test", tc.givenPrefix, appLister)
+			transformer := legacy.NewTransformer("test", tc.givenPrefix, appLister)
 			gotEvent, gotEventType := transformer.TransformLegacyRequestsToCE(writer, request)
-			wantEventType := formatEventType(tc.givenPrefix, tc.givenApplication, tc.givenEventName, tc.wantVersion)
+			wantEventType := fmt.Sprintf("%s.%s.%s.%s", tc.givenPrefix, tc.givenApplication, tc.givenEventName, tc.wantVersion)
```

Comment:
I think we need to bring the func `formatEventType` back, because it also handles the empty prefix.

Reply:
`formatEventType` is the only reason we need to have the test in the same module; removing it allows to move the tests to a `_test` module. And then test table does not contain an empty-prefix case.